site/app/models/gradeable/Gradeable.php

```diff
@@ -756,7 +760,7 @@ private function assertDates(array $dates) {
      * @param \DateTime[] $dates Array of dates, indexed by property name
      * @return \DateTime[] Array of dates, indexed by property name
      */
-    private function coerceDates(array $dates) {
+    private function coerceDates(array $dates, $regrade_modified = false) {
```

Comment:
Does this new parameter need a type?

Reply:
It definitely should. I'll go ahead and fix both.